examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,415 @@
+"""
+.. _ref_dcb_test_composite_delamination
+
+======================================================================
+Static simulation of double cantilever beam test via cohesive elements
+======================================================================
+
+This example is inspired to a classic double cantilever beam test commonly used
+to study mode I interfacial delamination of composite plates.
+
+Description
+===========
+
+Objective
+~~~~~~~~~
+
+This example shows how to use PyMAPDL to simulate delamination in
+composite materials. PyDPF modules are also used for the post-processing of the results.
+
+Problem Figure
+~~~~~~~~~~~~~~
+.. image:: ../../../images/dcb_test.png
+   :width: 400
+   :alt: DCB experimental test setup
+
+Procedure
+~~~~~~~~~
+* Launch MAPDL instance
+* Setup the model
+* Solve the model
+* Plot results of interest using PyMAPDL
+* Plot results using PyDPF
+* Plot reaction force
+
+Additional Packages Used
+~~~~~~~~~~~~~~~~~~~~~~~~
+* `Matplotlib <https://matplotlib.org>`_ is used for plotting purposes.
+* `Pandas <https://pandas.pydata.org/>`.
```

Comment:
```suggestion
* - `Matplotlib <https://matplotlib.org>`_ for plotting
* - `Pandas <https://pandas.pydata.org/>`_ for data analysis and manipulation
```

Reply:
Shouldn't NumPy be added to the list as well?

Reply:
Edits are being made to this PR to meet the Google Developer's Style Guide. An overall edit of this repo will happen soon.